elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "finos/morphir-elm",
     "summary": "Morphir Elm bindings",
     "license": "Apache-2.0",
-    "version": "15.0.0",
+    "version": "15.1.0",
```

Comment:
Can you please put the package publishing in a separate PR by moving it to a different branch?